File: online-judge-solutions/Codeforces/1656A.py
#https://codeforces.com/problemset/problem/1656/A
#Lang		:	Python 3.8
#Time		:	280 ms
#Memory         :	16000 KB
for _ in range(int(input())):
    n,m=int(input()),{}
    x=map(int,input().split())
    for i,v in enumerate(x):m[v]=i+1
    m=dict(sorted(m.items()))
    print(m[list(m)[0]],m[list(m)[-1]])

File: online-judge-solutions/Codeforces/1656A.cpp
//https://codeforces.com/problemset/problem/1656/A
//Lang		:	GNU C++14
//Time		:	62 ms
//Memory	:	3200 KB
#include<iostream>
#include<map>
using namespace std;
#define fastInp cin.tie(0); cout.tie(0); ios_base::sync_with_stdio(0);
int main(){
	fastInp;
	int t,n,x;
	cin>>t;
	while(t--){
		cin>>n;
		map<int,int> m;
		for(int i=0;i<n;i++){
			cin>>x;
			m[x]=i+1;
		}
		cout<<m.begin()->second<<" "<<m.rbegin()->second<<endl;
	}
	return 0;
}
